### PR TITLE
More ui updates

### PR DESCRIPTION
**Pull request by jaortiz**

### CODE REVIEW

File: website/dynamic-blocks/headings.js [hunk 1]
```diff
@@ -147,7 +147,7 @@ export const Heading = {
 								Heading: {
 									styles: (styles) => ({
 										...styles,
-										scrollMarginTop: '75px',
+										scrollMarginTop: '166px',
```
Comment (jonnystening) [inline website/dynamic-blocks/headings.js:150]: shouldnt this be in rem?

File: website/dynamic-blocks/vision-filters.js [hunk 2]
```diff
@@ -238,7 +238,7 @@ export const VisionFilters = {
 								Heading: {
 									styles: (styles) => ({
 										...styles,
-										scrollMarginTop: '75px',
+										scrollMarginTop: '166px',
```
Comment (jonnystening) [inline website/dynamic-blocks/vision-filters.js:241]: rem?

File: website/src/components/layout/sidebar.js [hunk 3]
```diff
@@ -32,9 +32,9 @@ export const Sidebar = ({ items }) => {
 				gridColumnStart: 1,
 				gridColumnEnd: 2,
 				borderRight: `1px solid ${COLORS.border}`,
+				width: '300px',
```
Comment (jonnystening) [inline website/src/components/layout/sidebar.js:35]: if we must have this, can we make it '100%' instead?

File: website/src/components/layout/sidebar.js [hunk 4]
```diff
@@ -32,9 +32,9 @@ export const Sidebar = ({ items }) => {
...
 				height: '100vh',
 				boxSizing: 'border-box',
-
```
Comment (jonnystening) [inline website/src/components/layout/sidebar.js:37]: 👍

File: website/src/components/layout/sidebar.js (hunk 3 above)
Comment (jaortiz) [reply website/src/components/layout/sidebar.js:35]: This is to fix that overhang that I'm getting. I can't figure out how else to hide it. 100% makes it even wider. This is the only way I have found that fixes it.

**Pull request merged**